### PR TITLE
New Event: waveform-ready

### PR DESCRIPTION
**Pull request by X-Raym**
Add event which triggers when waveforms are drawn, if media element
backend.

### CODE REVIEW

**Comment by X-Raym** [comment]
Initial discussion : https://github.com/katspaugh/wavesurfer.js/issues/736

@katspaugh Conflict resolved.

File: src/wavesurfer.js [hunk 1]
```diff
@@ -378,11 +378,11 @@ var WaveSurfer = {
      *  @param  {Array}            [peaks]     Array of peaks. Required to bypass
      *                                          web audio dependency
      */
-    loadMediaElement: function (urlOrElt, peaks) {
+    loadMediaElement: function (urlOrElt, peaks, preload) {
         var url = urlOrElt;
 
         if (typeof urlOrElt === 'string') {
-            this.backend.load(url, this.mediaContainer, peaks);
+            this.backend.load(url, this.mediaContainer, peaks, preload);
```
Comment (katspaugh) [inline src/wavesurfer.js]: It's showing your preload-related commits.

**Comment by katspaugh** [comment]
Please don't include `dist` into your PR. I'll build when releasing.

File: src/wavesurfer.js (hunk 1 above)
Comment (X-Raym) [inline src/wavesurfer.js]: @katspaugh hmm not sure how to avoid that. Maybe should I just create a simple patch as only one file need to be modified.

Comment (X-Raym) [inline src/wavesurfer.js]: @katspaugh How will you do that ?
Should I delete my forked repo and created and new one ?
Or should I just close this pull request and create a simple patch ?
I don't use advanced git collaborative functions very often 🤐

Comment (katspaugh) [inline src/wavesurfer.js]: Deleting a fork should work!

**Pull request closed without merging**